### PR TITLE
Magento 2 4 2 Beta Release Notes

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request (PR) adds release notes for the Beta versions of Magento Commerce and Open Source 2.4.2. 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/release-notes/bk-release-notes.html

whatsnew
Added 2.4.2 beta [release notes](https://devdocs.magento.com/guides/v2.4/release-notes/bk-release-notes.html).

### CODE REVIEW

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 1]
```diff
@@ -0,0 +1,1455 @@
...
+
+Although code for these features is bundled with quarterly releases of the Magento core code, several of these projects (for example, B2B, Page Builder, and Progressive Web Applications (PWA) Studio) are also released independently. Bug fixes for these projects are documented in the separate, project-specific release information that is available in the documentation for each project.
+
+## Highlights
+
+Look for the following highlights in this release.
+
+### Substantial security enhancements
+
+This release includes over  security fixes and platform security improvements. All security fixes have been backported to Magento 2.4.1-p1 and Magento 2.3.6-p1.
```
Comment (jeff-matthews) [inline src/guides/v2.4/release-notes/commerce-2-4-2.md:32]:
```suggestion
This release includes over 35 security fixes and platform security improvements. All security fixes have been backported to Magento 2.4.1-p1 and Magento 2.3.6-p1.
```

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 2]
```diff
@@ -0,0 +1,1455 @@
...
+
+*  Developers can now style content type output differently per viewport without using the `!important` directive.
+
+*  Content migrated to Page Builder is no longer padded with default inline styling.
+
+*  Row content type is no longer a required container.
+
+*  Predefined margins and paddings are no longer required for content types.
+
+*  The new mobile viewport switcher and viewport scope for form field values lets users and developers
```
Comment (jeff-matthews) [inline src/guides/v2.4/release-notes/commerce-2-4-2.md:87]: Incomplete sentence

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 3]
```diff
@@ -0,0 +1,1455 @@
...
+
+{% include release-notes/engcomm-2-4-2-issues.md %}
+
+### System requirements
+
+Our technology stack is built on PHP and MySQL. For more information, see [System Requirements]({{site.baseurl}}/system-requirements.html).
+
+### Installation and upgrade instructions
+
+You can install Magento Commerce 2.4.2 using Composer.
```
Comment (jeff-matthews) [inline src/guides/v2.4/release-notes/commerce-2-4-2.md:1449]:
```suggestion
You can install Magento Commerce 2.4.2 using [Composer]({{ page.baseur l}}/install-gde/composer.html).
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Nice work @jfrontain. Just a few minor suggestions.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 4]
```diff
@@ -0,0 +1,1455 @@
...
+
+*  Object storage and future extensibility
+
+*  [Storing media files]((https://docs-beta.magento.com/user-guide/system/data-import-product-images.html#method-3-import-images-with-remote-storage) on AWS S3
+
+Support for AWS S3 has been added to all modules including B2B, PageBuilder, and Adobe Stock Integration.
+
+### Magento Functional Testing Framework (MFTF)
+
+MFTF 3.2.0 is now available. This release introduces error tolerance in both tests and test suit generation.  Additional enhancements and bug fixes are described in the [Magento Functional Testing Framework Changelog](https://github.com/magento/magento2-functional-testing-framework/blob/develop/CHANGELOG.md).
```
Comment (jeff-matthews) [inline src/guides/v2.4/release-notes/commerce-2-4-2.md:191]: Should this be 3.2.1? See https://github.com/magento/magento2-functional-testing-framework/blob/3.2.1/CHANGELOG.md#321.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 5]
```diff
@@ -0,0 +1,1455 @@
...
+
+*  Added localization support across stores to support tasks such as changing languages, carts, and currencies. <!--- MC-37801-->
+
+*  The GraphQL schema has been enhanced to optimize product data retrieval for configurable products with many variants. <!--- MC-36138-->
+
+*  Integer type object IDs have been deprecated in favor of `uid` attributes of type ID. <!--- MC-36346-->
+
+*  Added the `staging` attribute to the `ProductInterface` and `CategoryInterface` to determine if a product is staged and to view its associated campaign information. <!--- MC-33897-->
+
+*  Added support for multiple wish lists. You can use GraphQL to [create]({{ page.baseurl }}/graphql/mutations/create-wishlist.html), [delete]({{ page.baseurl }}/graphql/mutations/delete-wishlist.html), and [rename]({{ page.baseurl }}/graphql/mutations/update-wishlist.html) wish lists as well as [move]({{ page.baseurl }}/graphql/mutations/move-products-to-wishlist.html) or [copy]({{ page.baseurl }}/graphql/mutations/copy-products-to-wishlist.html) items between them. <!--- MC-32356-->
```
[the remark this answers is not in the document]
Comment (jfrontain) [reply src/guides/v2.4/release-notes/commerce-2-4-2.md:111]: excellent suggestion -- I've moved those two sections immediately following the platform upgrades section. thanks.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md (hunk 1 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/commerce-2-4-2.md:32]: added -- thanks.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md (hunk 3 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/commerce-2-4-2.md:1449]: added link to both files. thanks.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md (hunk 4 above)
Comment (jfrontain) [reply src/guides/v2.4/release-notes/commerce-2-4-2.md:191]: thanks for catching that!  Changed in both files.

**Review by jeff-matthews — APPROVED** [review]

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**